conventions/documentation/create.md

```diff
@@ -6,53 +6,99 @@ parent: Conventions
 
 
 + Resources that can be created through the console have a primary button called: **Create [resource-name]**
++ Create buttons should appear in the top right corner
 + If the resource supports freely switching between form and YAML views, the editor switching radio controls should be shown
 + If the resources supports form editing (but not freely switching between form and YAML,) the form should include an **Edit YAML** secondary action in the top right corner for users to optionally switch to YAML creation
 + If a form exists, the form is shown by default
 + If multiple creation types exists, the primary create button changes to **Create** and includes a dropdown to present the options
 + When creating a resource from the summary table view page, the create resource form/yaml is presented in-page
 + If creating a resource from another context, such as the topology view, these forms can be surfaced in a modal
 + Resources should have samples available on creation. When first viewing the form, the sample panel should be closed by default. When first viewing the YAML creation, the sample panel should be open by default.
++ Upon creating a resource, the user should generally be taken into that newly created resource's details. An exception can be if the resource list is not surfaced at a top level and the user should be oriented to its location (Operands do this as their list is in the operator,) so the user is taken to the resource list.
 
```

Comment:
Taking a second look at this section as I'm reading the page. I think what would be easier to read is to have each of these bullets in context of the images below, and make this more of a summary section. It's difficult to visualize every part of the UI when they are formatted as bullets at the top of the section. I think you could call the top section Summary, and then within the images in the body provide more detail. That's one opinion.